### ISSUE TITLE
Interactive rebase is worded differently in different menus

### ISSUE THREAD

**Issue by jonstodle** [ForkIssues/TrackerWin#643]
When right-clicking a commit in the commit graph I get the following menu:

![image](https://user-images.githubusercontent.com/1719761/73656574-bdb1c300-4690-11ea-9752-846177ad65ce.png)

Which shows the option _Rebase '' to Here Interactively..._, but if I drag-and-drop a branch to another branch in the branch list in the left side panel I get the following menu:

![image](https://user-images.githubusercontent.com/1719761/73656696-ffdb0480-4690-11ea-8a23-f1a240432a06.png)

Which show the option _Rebase Interactively '' on ''..._.

In short, _Interactively_ changes position in the sentence and relative to the word _Rebase_.

Personally I prefer the second position, as it makes it easier to distinguish between the regular rebase and the interactive rebase when scanning through the menu.

**Comment by DanPristupov** [comment]
> Personally I prefer the second position

Just to be sure, do you mean you prefer `Rebase Interactively feature/branch to Here ...`?

**Comment by jonstodle** [comment]
Correct 👍

**Comment by DanPristupov** [comment]
Done in 1.45
